tests/unit/ConfigurationActions.spec.ts

```diff
@@ -9,43 +9,44 @@ describe('Configuration Actions', () => {
-  import.meta.env.VUE_APP_AUTH_API_URL = 'auth api url'
-  import.meta.env.VUE_APP_AUTH_API_VERSION = '/auth api version'
-  import.meta.env.VUE_APP_AUTH_WEB_URL = 'auth web url'
-  import.meta.env.VUE_APP_BUSINESSES_URL = 'businesses url'
-  import.meta.env.VUE_APP_BUSINESS_CREATE_URL = 'business create url'
-  import.meta.env.VUE_APP_BUSINESS_EDIT_URL = 'business edit url'
-  import.meta.env.VUE_APP_BUSINESS_FILING_LD_CLIENT_ID = 'business filing ld client id'
-  import.meta.env.VUE_APP_DASHBOARD_URL = 'dashboard url'
-  import.meta.env.VUE_APP_LEGAL_API_URL = 'legal api url'
-  import.meta.env.VUE_APP_LEGAL_API_VERSION_2 = '/legal api version 2'
-  import.meta.env.VUE_APP_PAY_API_URL = 'pay api url'
-  import.meta.env.VUE_APP_PAY_API_VERSION = '/pay api version'
-  import.meta.env.VUE_APP_REGISTRY_HOME_URL = 'registry home url'
-  import.meta.env.VUE_APP_SENTRY_DSN = 'sentry dsn'
-  import.meta.env.VUE_APP_SITEMINDER_LOGOUT_URL = 'siteminder logout url'
-  import.meta.env.VUE_APP_STATUS_API_URL = 'status api url'
-  import.meta.env.VUE_APP_STATUS_API_VERSION = '/status api version'
-  import.meta.env.VUE_APP_KEYCLOAK_AUTH_URL = 'keycloak url'
-  import.meta.env.VUE_APP_KEYCLOAK_REALM = 'keycloak realm'
-  import.meta.env.VUE_APP_KEYCLOAK_CLIENTID = 'keycloak clientid'
+  // create environment variable object for testing
+  const env = {
+    BASE_URL: '/business/',
+    VUE_APP_PATH: '/business',
+    VUE_APP_ADDRESS_COMPLETE_KEY: 'address complete key',
+    VUE_APP_AUTH_API_URL: 'auth api url',
+    VUE_APP_AUTH_API_VERSION: '/auth api version',
+    VUE_APP_AUTH_WEB_URL: 'auth web url',
+    VUE_APP_BUSINESSES_URL: 'businesses url',
+    VUE_APP_BUSINESS_CREATE_URL: 'business create url',
+    VUE_APP_BUSINESS_EDIT_URL: 'business edit url',
+    VUE_APP_BUSINESS_FILING_LD_CLIENT_ID: 'business filing ld client id',
+    VUE_APP_DASHBOARD_URL: 'dashboard url',
+    VUE_APP_LEGAL_API_URL: 'legal api url',
+    VUE_APP_LEGAL_API_VERSION_2: '/legal api version 2',
+    VUE_APP_PAY_API_URL: 'pay api url',
+    VUE_APP_PAY_API_VERSION: '/pay api version',
+    VUE_APP_REGISTRY_HOME_URL: 'registry home url',
+    VUE_APP_SENTRY_DSN: 'sentry dsn',
+    VUE_APP_SITEMINDER_LOGOUT_URL: 'siteminder logout url',
+    VUE_APP_STATUS_API_URL: 'status api url',
+    VUE_APP_STATUS_API_VERSION: '/status api version',
+    VUE_APP_KEYCLOAK_AUTH_URL: 'keycloak url',
+    VUE_APP_KEYCLOAK_REALM: 'keycloak realm',
+    VUE_APP_KEYCLOAK_CLIENTID: 'keycloak clientid'
+  } as any
 
   it('fetches and loads the configuration to the store', async () => {
     // mock window.location getters
     delete window.location
-    window.location = {
-      origin: 'http://localhost',
-      pathname: '/business/CP1234567',
-      search: '?accountid=2288'
-    } as any
+    Object.defineProperty(window, 'location', {
+      value: new URL('http://localhost/business/CP1234567?accountid=2288'),
+      configurable: true
+    })
```

Comment:
I don't think I needed to change this over but now this is consistent with the other UI.